### ISSUE TITLE
Make a way out of dead ends

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1319]
### TL;DR: Let's make it easier to _back out of_ dead ends rather than hand-wring/mind-read about how to _avoid them_.

Even though I initially raised the "dead end" issue a while back and still do believe it to be a problem (primarily in genes), I've changed my mind about how I think we should approach it. I no longer think that site search results is the best place to embed data peeking. Though data availability peeking may be useful elsewhere; I'm conflicted (https://github.com/monarch-initiative/monarch-app/issues/997).

In the end, I was underwhelmed with my "data peek" ideas for site search once they were materialized on screen. They made the search results look too much like _data_ and less like _search results_. It is a slippery slope that I better appreciate after all the recent UX we have done. 
#### 1. "Broaden investigation**

If the target page is not terribly data rich, provide a path to more data, if broader. Preferably with the use of both breadcrumbs and within the ontology tree view. For instance, this is an actual example with actual numbers that could help the user strategically broaden the inquiry.
<img width="735" alt="screen shot 2016-07-26 at 2 19 17 pm" src="https://cloud.githubusercontent.com/assets/3658863/17155920/fe83474c-533b-11e6-9fba-82ddf294a2cc.png">
#### 2. "YMABII" (You might also be interested in)

The tree viewer works for phenotypes and diseases, but for Gene pages, I think we need a different approach that draws on a combination of homology, pathways, and gene families. For these purposes, ignore the fact that there's probably homology data missing from this page.

<img width="994" alt="screen shot 2016-07-26 at 2 13 32 pm" src="https://cloud.githubusercontent.com/assets/3658863/17155747/4dd9b6f6-533b-11e6-9f4e-f4a5cf716fa6.png">
#### 3. "SRT" Searches Related To**

If the number of search results is too few, include suggested related searches that may list more results. This one is the hardest to execute in a meaningful way, but just putting it out there. For instance (note: ignore the data issues with the W2B duplicated disease; one should be a gene).

<img width="999" alt="screen shot 2016-07-26 at 3 01 20 pm" src="https://cloud.githubusercontent.com/assets/3658863/17157053/d6b1d1d8-5341-11e6-88fe-69406076b949.png">

Thoughts welcome...

**Comment by TomConlin** [comment]
I am a fan of always keeping the search field populated with whatever was put in it to get to wherever you are. In the case of no results, altering what is already there is easier than starting from scratch.

**Comment by jmcmurry** [comment]
Yes, excellent point Tom. Just to clarify, you're talking about the fact that we blank out the search box once search is executed, and this is annoying, right?

<img width="957" alt="screen shot 2016-07-28 at 11 47 04 am" src="https://cloud.githubusercontent.com/assets/3658863/17225274/26a4e932-54b9-11e6-8d4d-827646d0d195.png">

**Comment by TomConlin** [comment]
yep, could flavor it red when there are no results

**Comment by jmcmurry** [comment]
ps TLDR: 
- [ ] retain dead ends in phenotype or disease tree browser
- [ ] make dead end classes not clickable in the phenotype or disease tree browser
- [ ] remove dead ends from search results (? less emphatic about this part; see #1317)

There are loads of classes for which we have no data at all (not even a description) [Wilson disease (in other animals)](https://monarchinitiative.org/disease/OMIA:001071) has 5 leaf nodes of which 4 have no data whatsoever. This may be a bug with orthology representation but the app-facing behavior remains important to fix.

In the fullness of time we may also want to have a toggle for the hierarchy browser: hide / show classes with no data.

**Comment by jmcmurry** [comment]
I like this viz related to data richness. 
<img width="122" alt="screen shot 2016-11-08 at 9 57 43 am" src="https://cloud.githubusercontent.com/assets/3658863/20113196/35b6925e-a5a4-11e6-807c-bfbdd3ccaf1c.png">
<img width="1039" alt="screen shot 2016-11-08 at 9 57 36 am" src="https://cloud.githubusercontent.com/assets/3658863/20113197/35b6b2c0-a5a4-11e6-80cd-961e3afe72a6.png">
